src/main/java/org/jboss/tools/intellij/openshift/ui/helm/ChartIcons.java

```diff
@@ -26,28 +24,25 @@ public class ChartIcons {
   private static final Path BASE_PATH = Paths.get("images", "helm");
   private static final String HELM_ICON = "helm.png";
 
-  public static javax.swing.Icon getIcon(ChartVersions chart) {
-    return getIcon(chart.getName() + chart.getDescription());
+  public static Icon getHelmIcon() {
+    return IconManager.getInstance().getIcon(BASE_PATH.resolve(HELM_ICON).toString(), ChartIcons.class);
```

Comment:
get a deprecatio nwarining on this line : @Deprecated("Use getIcon(path, classLoader)")

Reply:
This can't be corrected before we move to `IC-2023.2`. See https://github.com/redhat-developer/intellij-openshift-connector/issues/732#issuecomment-1991848937